hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -257,6 +269,14 @@ public String getFrameworkHostAddress() {
     return hostAddress;
   }
 
+  public String getConfigServerHost() {
+    String configServerHost = System.getProperty("mesos.hdfs.config.server.host");
```

Comment:
I was originally following the getConfigServerPort() pattern with the system property override, but upon further inspection we don't need this at all. Updated connector provider accordingly.